### DISCUSSION TITLE
What software license to use for this project?

### DISCUSSION THREAD

**Discussion by manuGil** [Citizens-Collective/Citizen-Voice#4] [Q&A]
Under the current [guidelines or research software](https://d2k0ddhflgrk1i.cloudfront.net/TUDelft/Over_TU_Delft/Strategie/TU%20Delft%20Research%20Software%20Guidelines.pdf), TU Delft encourages the use of open-source licenses for research software, and grant IP rights to the researchers and developers. 

TU Delft recommends, one of the followings **pre-approved** licences:

[![License: CC0-1.0](https://img.shields.io/badge/License-CC0%201.0-lightgrey.svg)](http://creativecommons.org/publicdomain/zero/1.0/)

[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)

[![License](https://img.shields.io/badge/License-BSD%203--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause)

[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)

[EUPL-1.2](https://opensource.org/licenses/EUPL-1.2)

[![License: GPL v2](https://img.shields.io/badge/License-GPL%20v2-blue.svg)](https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html)

[![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0)

[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0)

@cforgaci @trivikverma @jgves **What license do you prefer?** If you're not sure, we can consult the data stewards

**Comment by trivikverma** [comment]
I'm not sure. I'd prefer something that doesn't allow this to be sold for
profit but still allows people to extend it.

**Reply by manuGil** [reply]
Is it an idea to invite a data steward to this discussion so that he/she can help us to choose a license?

**Reply by jgves** [reply]
I can contact Nicolas at TPM

**Reply by manuGil** [reply]
Thanks @jgves  I think that will be the best in this case.

**Comment by trivikverma** [comment]
Thanks Juliana. I only have that one concern - the rest is fine for me.

**Comment by manuGil** [comment]
We will go for the GPL-v3 license as agreed.

We should complete the following info as part of TU Delft copyright claim. Could you please help me with the missing info (between brackets)?

```
Technische Universiteit Delft hereby disclaims all copyright interest in the program “Citizen-Voice” 
(an inclusive, web-based software platform for collaborative data collection that facilitates citizen participation) 
written by the Author(s). 
[Name Dean], Dean of [Name Faculty].

(c) 2022, [AUTHOR(S)], [REFERENCE TO PROJECTS, GRANTS OR STUDY (if desired)], Delft, The Netherlands.

This work is licensed under a GPLv3 OSS license.
```
